app/uk/gov/hmrc/exports/controllers/NotificationsController.scala

```diff
@@ -54,7 +56,19 @@ class NotificationsController @Inject()(
   def saveNotification(): Action[NodeSeq] = Action.async(parse.xml) {
     implicit request =>
       metrics.startTimer(notificationMetric)
-      validateHeaders(getNotificationFromRequest _ andThen save _)
+      implicit val body: NodeSeq = request.body
```

Comment:
Where you need this? I'm not sure but you can remove it

Reply:
yep will do